### DISCUSSION TITLE
Shadow on non focused items

### DISCUSSION THREAD

**Discussion by marcplanas10** [ImaginativeShohag/Why-Not-Image-Carousel#105] [Ideas]
Would be interesting if we could add a shadow to the items not selected in order to create a nice enviroment with the cards.

**Comment by ImaginativeShohag** [comment]
You can add any custom layout to the library. So you can do whatever you want :). Please check the [sample](https://github.com/ImaginativeShohag/Why-Not-Image-Carousel/tree/master/sample).

**Reply by marcplanas10** [reply]
yeah, but how you change this custom layout depending if the element is the one in the middle or not ?

**Reply by ImaginativeShohag** [reply]
It can be possible because you have the `item` and `position` in the `onBindViewHolder` of the carouse. You can use `carousel.currentPosition` to get the current item position. Using those you can add/remove shadow.
